### ISSUE TITLE
Cannot read properties of undefined (reading 'statusCode')

### ISSUE THREAD

**Issue by zeeklog** [TryGhost/Ghost-CLI#1912]
ghost update

Love open source? We’re hiring JavaScript Engineers to work on Ghost full-time.
https://careers.ghost.org

+ sudo systemctl is-active ghost_xxxx-com
✔ Checking system Node.js version - found v18.18.0
✔ Ensuring user is not logged in as ghost user
✔ Checking if logged in user is directory owner
✔ Checking current folder permissions
✔ Checking folder permissions
✔ Checking file permissions
✔ Checking content folder ownership
✔ Checking memory availability
✔ Checking free space
✔ Checking for available migrations
✖ Checking for latest Ghost version
An error occurred.
Message: 'Cannot read properties of undefined (reading 'statusCode')'

Debug Information:
    OS: Ubuntu, v22.04.4 LTS
    Node Version: v18.18.0
    Ghost Version: 5.72.0
    Ghost-CLI Version: 1.26.1
    Environment: production
    Command: 'ghost update'

**Comment by KarelWintersky** [comment]
this issue must be closed too ?

**Comment by zeeklog** [comment]
> [@zeeklog](https://github.com/zeeklog) are you running Ghost-CLI behind a proxy? if so this is likely a duplicate of [#1899](https://github.com/TryGhost/Ghost-CLI/issues/1899), which has been fixed in main but hasn't been released yet.

Yes, I'm running Ghost-CLI behind a proxy. I disabled update check by modifying the source code.

**Comment by ErisDS** [comment]
I think this is fixed and released now, so closing
